vmm/src/seccomp_filters.rs

```diff
@@ -236,6 +246,10 @@ pub fn vmm_thread_filter() -> Result<SeccompFilter, Error> {
             allow_syscall(libc::SYS_fsync),
             #[cfg(target_arch = "x86_64")]
             allow_syscall(libc::SYS_ftruncate),
+            #[cfg(target_arch = "aarch64")]
+            // The definition of libc::SYS_ftruncate is missing on AArch64.
```

Comment:
Please open an issue on the libc crate about this if there isn't one already.

Reply:
I created one for it: https://github.com/rust-lang/libc/issues/1803